### DISCUSSION TITLE
How do I set up an emergency fund?

### DISCUSSION THREAD

**Discussion by cbirkenbeul** [ghostfolio/ghostfolio#2721] [Q&A]
Hi,
i don't quite get it how to setup the emergency fund. Are there some documentation, that I did not find?
Thanks.

**Comment by dtslvr** [comment]
Hi @cbirkenbeul

You can define your emergency fund via _Home_ > _Summary_ > _Emergency Fund_. This amount is then listed separately and influences your buying power.

**Reply by cbirkenbeul** [reply]
Is that in the self hosted version too? Because I cannot find that.

**Reply by erikvanderwerf** [reply]
I also cannot find this on the self-hosted version.

**Reply by dtslvr** [reply]
Can you click here?

<img width="488" alt="emergency-fund-in-summary" src="https://github.com/ghostfolio/ghostfolio/assets/4159106/29ac75ce-3696-4e5f-a785-13ad2bfdb8c1">

**Reply by erikvanderwerf** [reply]
I see that now, I would never have noticed that on my own 😅

**Reply by dtslvr** [reply]
Great! The self-hosted version actually has the same features as the app running in the cloud.
